### PR TITLE
4.2.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.2.0 (2020-07-21)
+
+- Added ability to hide restricted items in the sidebar navigation. [#79](https://github.com/blackbaud/skyux-docs-tools/pull/79)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: Can you add "the" before ability? Otherwise looks good!

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.2.0-release](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.2.0-release)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: done!

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.2.0-release](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.2.0-release)

**Pull request merged**